docs/litmus-admin-rbac.yaml

```diff
@@ -14,9 +14,12 @@ metadata:
   labels:
     name: litmus-admin
 rules:
-- apiGroups: ["","apps","batch","extensions","litmuschaos.io","openebs.io","storage.k8s.io"]
-  resources: ["chaosengines","chaosexperiments","chaosresults","cstorpools","cstorvolumereplicas","configmaps","secrets","pods","pods/exec","pods/log","pods/eviction","jobs","replicasets","deployments","daemonsets","statefulsets","persistentvolumeclaims","persistentvolumes","storageclasses","services","events"]
+- apiGroups: ["","apps","batch","extensions","litmuschaos.io"]
+  resources: ["pods","pods/exec","pods/eviction","jobs","daemonsets","events","chaosresults","chaosengines"]
   verbs: ["create","delete","get","list","patch","update", "deletecollection"]
+- apiGroups: ["","apps","litmuschaos.io"]
+  resources: ["pods","configmaps","secrets","events","services","chaosexperiments","pods/log","replicasets","deployments","statefulsets",services]
+  verbs: ["get","list","patch","update"]
 - apiGroups: [""]
   resources: ["nodes"]
```

Comment:
why we need patch and update on nodes

Reply:
Experiments like Node Taint needs that 

ref: https://github.com/litmuschaos/litmus-go/blob/0fa4a508f9c67241dbb28d3b60da1588582bcfcc/chaoslib/litmus/node-taint/lib/node-taint.go#L198